source/12-troubleshooting.md

```diff
@@ -3,7 +3,7 @@ title: 12. 排错
 description: 当你遇到问题的时候应该看看这个
 type: docs
 author_info: 由 xMuu、Sukka 整理，由 Sukka 翻译
```

Comment:
Would you mind adding your name to it?

E.g.

```suggestion
author_info: 由 xMuu、Sukka、	derbalkon 整理，由 Sukka、derbalkon 翻译
```

So does the other pages.

Reply:
Right, totally forgot! Will do.